yt_idv/scene_data/block_collection.py

```diff
@@ -140,3 +140,17 @@ def _load_textures(self):
             )
             self.texture_objects[vbo_i] = data_tex
             self.bitmap_objects[vbo_i] = bitmap_tex
+
+    _grid_id_list = None
+
+    @property
+    def grid_id_list(self):
+        # get the grids that contain all the blocks
+        if self._grid_id_list is None:
+            gl = [gid for gid, _ in self.grids_by_block.values()]
```

Comment:
Check that you don't need to do a `_id_offset` for these -- sometimes we do for grids that are indexed-by-1 as opposed to indexed-by-0.  I suspect this is fine, but I think we need to double check.

Reply:
Ooook, up where `grids_by_block` is built, it stores `g.id - g._id_offset`, so this `grid_id_list` will return the 0-indexed grid id's. So I'm pretty sure this is correct (I'll add a comment here though). I did also try one of the yt flash sample datasets  that uses a nonzero `_id_offset` and the grid outline looked correct to me.